docs/man-openmpi/man3/MPI_Session_call_errhandler.3.rst

```diff
@@ -72,9 +72,6 @@ processes in session if the default error handler has not been changed.
 Errors
 ------
 
-Almost all MPI routines return an error value; C routines as the value
-of the function and Fortran routines in the last argument. See the MPI
-man page for a full list of MPI error codes.
-
+.. include:: ./ERRORS.rst
```

Comment:
Given the nature of this call, I think the original text is all that is needed.

Reply:
I put back the reference to the MPI man page, but I think the new error text also applies here.